pages/content.html

```diff
@@ -0,0 +1,161 @@
+  <li>countering</li>
+  <li>deliver (pizzas, mail, and services are delivered – not abstract concepts like “improvements” or “priorities”)</li>
+  <li>deploy (unless it's military or software)</li>
+  <li>dialogue (we speak to people)</li>
+  <li>facilitate (instead, say something specific about how you are helping)</li>
+  <li>focusing</li>
+  <li>foster (unless it's children)</li>
+  <li>impact (as a verb)</li>
+  <li>initiate</li>
+  <li>key (unless it unlocks something. A subject/thing isn’t “key” – it’s probably “important”)</li>
+  <li>land (as a verb, only use if you are talking about aircraft)</li>
+  <li>leverage (unless in the financial sense)</li>
+  <li>overarching</li>
+  <li>progress (as a verb – what are you actually doing?)</li>
+  <li>promote (unless you are talking about an ad campaign or some other marketing promotion)</li>
+  <li>robust</li>
+  <li>streamline</li>
+  <li>strengthening (unless it’s strengthening bridges or other structures)</li>
+  <li>tackling (unless it's rugby, football or some other sport)</li>
+  <li>transforming (what are you actually doing to change it?)</li>
+  <li>utilize</li>
+</ul>
+
+<p>Always avoid metaphors. For example:</p>
+
+<ul>
+  <li>drive (you can only drive vehicles; not ideas or people)</li>
+  <li>going forward (unlikely we are giving travel directions)</li>
+  <li>in order to (superfluous – don’t use it)</li>
+  <li>one-stop shop (we are government, not a retail outlet)</li>
+</ul>
+
+<p>With all of these words you can generally get rid of them by breaking the term into what you’re actually doing. Be open and specific.</p>
+
+<p>Write conversationally – picture your audience and write as if you were talking to them one-to-one but with the authority of someone who can actively help. All audiences should understand our content. This isn’t “dumbing down;” this is opening up government information to all.</p>
+
+<h2>Publishing legal and technical content</h2>
+
+<p>Legal content can still be written in plain English. It's important that users understand content and that we present complicated information simply. Where evidence shows there's a clear user need for including a legal term, always explain it in plain English. Where you need to use technical terms, you can. They're not jargon. You just need to explain what they mean the first time you use them.</p>
+
+<h2>Websites</h2>
+
+<p>Websites shouldn’t include “http://” at the beginning or “/” at the end.</p>
+
+<p>Data.gov should be capitalized accordingly.</p>
+
+<h2>Help improve this content</h2>
+
+<p>Content suggestions are welcome, although we can’t guarantee your changes will be included. To suggest inline edits, simply log in to GitHub, and click the pencil icon of <a href="https://github.com/GSA/data.gov-styleguide/blob/gh-pages/index.md">this file</a>, or to comment, open a New Issue <a href="https://github.com/GSA/data.gov-styleguide/issues">here</a>.</p>
```

Comment:
Making a call for this product to use 18F's content style guide (very similar to the content style guide suggested by @ErieMeyer ):
https://pages.18f.gov/content-guide/faqs/

Currently working to figure out if it makes sense to surface the content style guide directly on our site or link to 18F for this. Stay tuned -- expecting resolution shortly.